hrm-service/src/case/case.ts

```diff
@@ -16,7 +16,7 @@ import {
 import { randomUUID } from 'crypto';
 import { Contact } from '../contact/contact-data-access';
 import { setupCanForRules } from '../permissions/setupCanForRules';
-import { User } from '@tech-matters/twilio-worker-auth';
+import { TwilioUser } from '@tech-matters/twilio-worker-auth';
```

Comment:
I honestly don't know what the benefit of using the type keyword on individual imported symbols is. If you are doing any 'non type' imports from that module you'll have to do a 'real' import on the full module anyway?

I'm going to keep missing these because I let Webstorm take care of imports and it doesn't seem smart enough to recognise when it can be a type import. Maybe it's because our version of prettier doesn't like it

Reply:
Ahh yes. We had that lull in HRM development when a prettier upgrade would have made sense and I forgot about it. Now that we have a bunch of work to do it raises its head again. c'est la vie. 

I added a note to my personal trello board.  Maybe I'll remember next time there is a lull... maaaybe.